bundles/net.tourbook.export/src/net/tourbook/export/TourExporter.java

```diff
@@ -987,12 +1000,26 @@ public boolean export(final String exportFileName) {
 
       doExport_70_WayPoints(wayPoints, tourMarkers, trackStartTime);
 
+      boolean exportStatus = false;
       try {
-         return doExport_10_Tour(tracks, wayPoints, tourMarkers, tourLap, exportFileName);
+
+         exportStatus = doExport_10_Tour(tracks, wayPoints, tourMarkers, tourLap, exportFileName);
+
       } catch (final IOException e) {
+
          StatusUtil.log(e);
       }
-      return false;
+
+      if (_displayNotificationPopup) {
+
+         final String notificationText = exportStatus
+               ? "\u2713 " + String.format(Messages.Dialog_Export_Message_Successful, exportFileName) //$NON-NLS-1$
```

Comment:
Symbols should use a constant with a similar name which is displayed in the dialog, e.g.

![image](https://github.com/mytourbook/mytourbook/assets/1283445/d0386888-1e49-4283-8699-d8ffd96a3c58)

![image](https://github.com/mytourbook/mytourbook/assets/1283445/f6267b3f-bc6d-45fb-93a8-2a232ee2782a)

Reply:
Fixed